applications/GeoMechanicsApplication/tests/test_set_multiple_moving_load_process.py

```diff
@@ -0,0 +1,1886 @@
+import KratosMultiphysics
+
+import KratosMultiphysics.GeoMechanicsApplication as GMA
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+class TestSetMultipleMovingLoadsProcess(KratosUnittest.TestCase):
```

Comment:
I have some suggestions to improve this class. Let's discuss this in the office. Also SonarCloud does not agree with this change.